### DISCUSSION TITLE
How to display Create Resource Page from modal

### DISCUSSION THREAD

**Discussion by aditnanda** [filamentphp/filament#6664] [Help]
I want to display or call my Resource Create for example "CreateUser" in the modal that I created in the livewire component. Is there a way to display it using a modal without redirecting to the resource create page? Thank You

**Comment by danharrin** [comment]
We have a built in modal to do this for you. You just need to delete the Create page class and remove it from `getPages()`.

**Reply by dayemsiddiqui** [reply]
@danharrin 
This approach doesnt work if someone wants to override the method in the CreateResource page:
e.g

```
<?php

namespace App\Filament\Resources\ClientResource\Pages;

use App\Filament\Resources\ClientResource;
use Filament\Resources\Pages\CreateRecord;

class CreateClient extends CreateRecord
{
    public static $openInNewTab = false;

    // Open this page in a slide-over
    public static $showPageTitle = true;

    // Show the page title
    protected static string $resource = ClientResource::class;

    protected function mutateFormDataBeforeCreate(array $data): array
    {
        $data['user_id'] = auth()->id();
        return $data;
    }


}
```

There should be a programmatic way in the CreateResource page to open it in a modal

**Reply by danharrin** [reply]
You do not need to override methods on the page if you are using a modal, every method has an equivalent that exists on the Action class that can be used to configure it. Check the docs
